app/controllers/questions_controller.rb

```diff
@@ -11,6 +11,10 @@ def create
     ### call NEW/CREATE method for USER_QUESTION
   end
 
+  def update
```

Comment:
are we allowing users to update a question? or should they just create a new one? or can just admin update a question?